index.html

```diff
@@ -1392,14 +1414,14 @@ <h2>The <dfn>NDEFRecordType</dfn> enum</h2>
   <a>Web NFC message</a>s either from an <a>NFC tag</a> or an
   <a>NFC peer</a>.
   <pre class="idl">
-    typedef (DOMString or ArrayBuffer or NDEFMessage) NDEFMessageSource;
+    typedef (DOMString or ArrayBuffer or NDEFMessageInit) NDEFMessageSource;
 
     [Constructor(), SecureContext, Exposed=Window]
     interface NFCWriter {
-      Promise&lt;void&gt; push(NDEFMessageSource message, optional NFCPushOptions options);
+      Promise&lt;void&gt; push(NDEFMessageSource message, optional NFCPushOptions options={});
     };
 
-    [Constructor(optional NFCReaderOptions options), SecureContext, Exposed=Window]
+    [Constructor(optional NFCReaderOptions options={}), SecureContext, Exposed=Window]
```

Comment:
if you add ={} I don't think you need the optional any longer?

Reply:
This was a new ReSpec error:
`WebIDL validation error: Optional dictionary arguments must have a default value of {}`

Do you have any solution?

Reply:
OK, maybe that is new, let's see what @marcoscaceres says

Reply:
Yeah, you still need the optional :)